Fix some issues in the Calling Java Code from Ballerina guide (#1823)
ballerina-platform/ballerina-dev-website: 1.2/learn/calling-java-code-from-ballerina.md

>**Note:** If a Java exception class is generated as a Ballerina binding object, it would follow the naming convention `JException` or `JError`. For instance, the binding object's name for `java.io.FileNotFoundException` would be as `JFileNotFoundException`.

## Packaging Java Libraries with Ballerina Programs

> This section assumes that you have already read [Structuring Ballerina Code](/1.2/learn/structuring-ballerina-code/). When you compile a Ballerina program with `ballerina build <root-module>`, the compiler creates an executable JAR file and when you compile a Ballerina module with `ballerina build -c <module>`, the compiler creates a BALO file. In both cases, the Ballerina compiler produces self-contained archives. There are situations in which you need to package JAR files with these archives. The most common example would be packing the corresponding JDBC driver.

Reviewer: ```suggestion

>**Tip:** As a prerequisite for this section, see [Structuring Ballerina Code](/1.2/learn/structuring-ballerina-code/). 
When you compile a Ballerina program with `ballerina build <root-module>`, the compiler creates an executable JAR file and when you compile a Ballerina module with `ballerina build -c <module>`, the compiler creates a BALO file. In both cases, the Ballerina compiler produces self-contained archives. There are situations in which you need to package JAR files with these archives. The most common example would be packing the corresponding JDBC driver.
```

Now, let’s briefly look at how a Java exception is converted to a Ballerina error value at runtime. A Ballerina error value contains three components: a reason, a detail, and stack trace. 

The `reason`:

> - This is a string identifier for the category of error.

Reviewer: ```suggestion
- This is a string identifier for the error category.
```
